### ISSUE TITLE
Giveaway details question

### ISSUE THREAD

**Issue by KNYuki** [imaboy321/Ledybot#29]
Hi.  Question, is it possible to (or implement) mass delete of a column, say the "Specific Path" column in my case? It's not so much an issue but an inconvenience when I want to trade nicknamed deposits only, but to do so I need the specific path entries to be empty - unless I'm doing this wrong? Otherwise the bot will trade anyone who makes the request in the GTS. Thing is, if I'm mass importing by the folders, having to manually clear hundreds of fields one by one gets tiring lol.  So yeah, unless I'm doing it wrong, being able to edit multiple fields at once would be nice.

**Comment by gr33nd3v1l** [comment]
I hope I'm understanding the issue correctly, but when I import my files, I make sure the Ditto file is named Ditto.pk7 so it automatically creates a Ditto folder, in which i put the Adamant.pk7 and others

Trading has worked for me like this,  if I remember correctly

**Comment by KNYuki** [comment]
I have mine named as something like "132 - Ditto.pk7". Similarly, when I import it, it'll create a folder with the same name where I too put the nature .pk7s into.  

The problem I have with this is, when I drag and drop by the folder instead of individual .pk7s, the giveaway details fill in automatically where specific path is "132 - Ditto.pk7" and optional path leads to the Ditto folder, this ends up trading _everyone_ who requests for a ditto (nicknamed or not)  instead of only trading the nicknamed ones.  Clearing the specific path field helped fixed that, but my real problem is clearing the field one by one for hundreds of entries.

**Comment by Souljahx6** [comment]
I been trying to figure this out for 2 days. I do giveaways and I ask for them to nickname Gastly Psamar and deposit. I want ledy bot to only trade to those that nicknamed the gastly but it always ends up trading everyone. I removed the dafult pk7 and renamed the pk7 in the Castform folder Psamar.pk7 and it still doesn't work. here is an image of my setup.  Any ideas? 

I have tried every which way. 

![2017-05-18_21-26-45](https://cloud.githubusercontent.com/assets/8357047/26229643/5316fdc6-3c11-11e7-8dd5-6920593e4169.png)
